device/tt_silicon_driver.cpp

```diff
@@ -531,6 +535,22 @@ tt_SiliconDevice::tt_SiliconDevice(const std::string &sdesc_path, const std::str
             }
         }
     }
+
+    // Default initialize host_address_params based on detected arch
+    switch(arch_name) {
+        case tt::ARCH::GRAYSKULL:
+            host_address_params = {grayskull::host_mem::address_map::ETH_ROUTING_BLOCK_SIZE, grayskull::host_mem::address_map::ETH_ROUTING_BUFFERS_START};
+            break;
+
```

Comment:
This may give the reader the impression that the notions of ETH block size and ETH routing buffer have meaning for Grayskull.  They do not.

Reply:
Would a comment to that effect suffice?
I can't really hardcode it to 0x0, 0x0 because we stuff an actual value for ETH_ROUTING_BLOCK_SIZE in grayskull.

Reply:
Yes, a comment is fine.